src/main.zig

```diff
@@ -142,6 +145,53 @@ fn help(name: []const u8, args: *std.process.ArgIterator, allocator: std.mem.All
     try std.io.getStdOut().writer().print("\n", .{});
 }
 
+fn init() Command_Error!void {
+    // not experienced in zig enough to try this but i feel like you could initalize
+    // both at the same time and in the switch(i) check if both files exist and only if
+    // both conditions are false, continue with creating the rest of the files
+    var main_orng = try std.fs.cwd().createFile("main.orng", .{ .exclusive = true}) catch |i| {
+    switch (i) {
+        i.PathAlreadyExists => {
+            std.log.err("Main.orng already present.", .{});
+            return 1;
+        }, else => {
+            std.log.err("Unable to initalize file main.orng", .{}); // Seeing if it's able to create the file; if not, return error
+            return 1;
```

Comment:
In the case of an unknown file create error, I think it would be best to bubble up whatever error was received up back to `main()`, so this prong would look something like `else => return i,`